src/preset-themes/001.json

```diff
@@ -0,0 +1,35 @@
+{
+  "title": "001",
```

Comment:
Threw in a title for the heck of it. Filenames & titles are 🤷‍♂️ because I wasn't feeling creative. We can tweak.